db/db_iter.cc

```diff
@@ -721,14 +722,17 @@ bool DBIter::ReverseToBackward() {
         saved_key_.GetUserKey(), kMaxSequenceNumber, kValueTypeForSeek));
     if (!expect_total_order_inner_iter()) {
       iter_.SeekForPrev(last_key.GetInternalKey());
+      RecordTick(statistics_, NUMBER_OF_RESEEKS_IN_ITERATION);
     } else {
       // Some iterators may not support SeekForPrev(), so we avoid using it
       // when prefix seek mode is disabled. This is somewhat expensive
       // (an extra Prev(), as well as an extra change of direction of iter_),
       // so we may need to reconsider it later.
       iter_.Seek(last_key.GetInternalKey());
+      RecordTick(statistics_, NUMBER_OF_RESEEKS_IN_ITERATION);
       if (!iter_.Valid() && iter_.status().ok()) {
         iter_.SeekToLast();
+        RecordTick(statistics_, NUMBER_OF_RESEEKS_IN_ITERATION);
```

Comment:
We have the same pattern (`Seek` potentially followed by `SeekToLast` if we went too far) in `FindValueForCurrentKeyUsingSeek`; I'm wondering if we should bump the counter twice there as well? (Or just once here too to be consistent with that.)

Reply:
Good catch. I do not have a strong opinion about this. Since the counter values may be used by applications already, I suggest that I keep it consistent with existing pattern to reduce the scope of this PR. We can solve the issue you mentioned in a separate PR.